dvc/rwlock.py

```diff
@@ -9,7 +9,7 @@
 
 from .exceptions import DvcException
 from .lock import LockError
-from .utils.fs import relpath
+from .utils import relpath
```

Comment:
Are we trying to move this?

Reply:
@Suor It is a faulty import from an incorrect place that broke once I've removed no longer used `relpath` re-import from fs.py.